runtime/queries/purescript/textobjects.scm

```diff
@@ -0,0 +1,13 @@
+(comment) @comment.inside
+
+[
+  (decl_adt)
```

Comment:
It looks like this is failing the query check since the node isn't in the grammar